snippets/standard/buffers/memory-t/task-returning/task-returning.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <Snippet1>
+using System;
+using System.Buffers;
+using System.IO;
+using System.Threading.Tasks;
+
+public class Example
+{
+    // <Snippet1>
+    // An acceptable implementation.
+    static void Log(ReadOnlyMemory<char> message)
```

Comment:
I'd prefer seeing this method return the `Task` so that it can be awaited.

Unless, what you're showing is how `ReadOnlyMemory` is limited in it use because it is a `ref struct`. I didn't compile this code, but I think this use is allowed, but the caller may be limited.